app/models/ems_event.rb

```diff
@@ -72,6 +72,10 @@ def self.add_openstack(ems_id, event)
     add(ems_id, ManageIQ::Providers::Openstack::CloudManager::EventParser.event_to_hash(event, ems_id))
   end
 
+  def self.add_openstack_network(ems_id, event)
+    add(ems_id, ManageIQ::Providers::Openstack::NetworkManager::EventParser.event_to_hash(event, ems_id))
+  end
```

Comment:
Interesting, let me do that in a separate PR, I think at some point, I will be separating OpenStack to a gem too. :-)